src/pages/components/notification/code.mdx

```diff
@@ -4,66 +4,142 @@ description: Notifications are messages that communicate information to the user
 tabs: ["Usage", "Style", "Code", "Accessibility"]
 ---
 
-<ComponentDemo
-  components={[
-    {
-      id: "toast",
-      label: "Toast notification",
-    },
-    {
-      id: "inline",
-      label: "Inline notification",
-    },
-  ]}
->
-  <ComponentVariant
-    id="toast"
-    knobs={{
-      ToastNotification: ["hideCloseButton", "kind", "lowContrast"],
-    }}
-    links={{
-      React:
-        "https://react.carbondesignsystem.com/?path=/story/notifications--toast",
-      Angular:
-        "https://angular.carbondesignsystem.com/?path=/story/components-notification--toast",
-      Vue:
-        "http://vue.carbondesignsystem.com/?path=/story/components-cvtoastnotification--default",
-      Vanilla: "https://the-carbon-components.netlify.com/?nav=notification",
-    }}
-  >{`
-    <div>
-      <ToastNotification
-        caption="00:00:00 AM"
-        iconDescription="describes the close button"
-        subtitle={<span>Subtitle text goes here. <a href="#example">Example link</a></span>}
-        timeout={0}
-        title="Notification title"
-      />
-    </div>
-`}</ComponentVariant>
-  <ComponentVariant
-    id="inline"
-    knobs={{
-      InlineNotification: ["hideCloseButton", "lowContrast"],
-    }}
-    links={{
-      React:
-        "https://react.carbondesignsystem.com/?path=/story/notifications--inline",
-      Angular:
-        "https://angular.carbondesignsystem.com/?path=/story/components-notification--basic",
-      Vue:
-        "http://vue.carbondesignsystem.com/?path=/story/components-cvinlinenotification--default",
-      Vanilla: "https://the-carbon-components.netlify.com/?nav=notification",
-    }}
-  >{`
-  <div>
+<PageDescription>
+
+How to build a notification using React.
+
+</PageDescription>
+
+<AnchorLinks>
+
+<AnchorLink>Overview</AnchorLink>
+<AnchorLink>Component API</AnchorLink>
+<AnchorLink>Feedback</AnchorLink>
+
+</AnchorLinks>
+
+## Overview
+
+Carbon Notifications come in two variants: toast, and inline. They are created by using the `ToastNotification` or `InlineNotification` components.
+
+## Toast notification
+
+You can use the `ToastNotification` component to display a non-modal, time-based short message that appears at the bottom or the top of the screen.
+
+```jsx
+import { ToastNotification } from "carbon-components-react";
+
+function MyComponent() {
+  return (
+    <ToastNotification
+      caption="00:00:00 AM"
+      iconDescription="describes the close button"
+      subtitle={
```

Comment:
Would we want to demonstrate the text-based prop instead of the custom node use-case? Trying to think about the simplest example here, we could always show this example as a custom node variant later on too.